.circleci/config.yml

```diff
@@ -43,6 +43,20 @@ jobs:
           paths:
             - venv
             - unpacked_sdist
+  run_pp:
+    working_directory: ~/featuretools
+    parameters:
+      image_tag:
+        type: string
+        default: "3.7.3"
+    executor:
+      name: python
+      image_tag: << parameters.image_tag >>
+    steps:
+      - checkout
+      - run:
+          name: "Triger PP tests"
+          command: "curl -u ${PP_K}: -d build_parameters[CIRCLE_JOB]=python-35 https://circleci.com/api/v1.1/project/github/FeatureLabs/premium-primitives/tree/master"
```

Comment:
This job config can be simplified a lot.  Look at the changelog_updated job as a reference.  The image can just be busybox and it might not even require running the checkout command

Reply:
Actually, busybox doesn't have curl, only wget

Reply:
Using circleci-buildpack, curl image